integration/analytics/Tiltfile

```diff
@@ -1,5 +1,9 @@
 # -*- mode: Python -*-
 
+load_namespace = not str(local('echo -n $SKIP_NAMESPACE'))
```

Comment:
can you add comments here in all the Tiltfiles so future devs know it's part of the infra stuff and must be in every integ. test Tiltfile, rather than it being specific to this test?